### PR TITLE
[release][website] Add release notes for Release 2.1.0

### PR DESCRIPTION
**Pull request by sijie**

### CODE REVIEW

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
 
 ## Apache incubator
 
+### 2.0.0-rc1-incubating &mdash; 2018-05-29 <a id="2.0.0-rc1-incubating"></a>
```
Comment (merlimat) [inline site2/website/release-notes.md]: this should be 2.1.0

Comment (sijie) [reply site2/website/release-notes.md]: ah, your are right. I forgot to change it after copying from 2.0.0 :)

Comment (sijie) [reply site2/website/release-notes.md]: fixed

**Comment by sijie** [comment]
@merlimat  I have fixed the headline

**Comment by sijie** [comment]
retest this please

**Pull request merged**